src/backend/aspen/workflows/nextstrain_run/export_test.py

```diff
@@ -68,12 +102,24 @@ def cli(
             "num_sequences": num_sequences,
             "num_included_samples": num_included_samples,
         }
-        group = session.query(Group).first()
+        if group_name:
+            group = session.query(Group).filter(Group.name == group_name).first()
+        else:
+            group = session.query(Group).first()
+        if not group:
+            raise Exception("No group found")
+        if location:
+            (region, country, div, loc) = location.split("/")
+            tree_location = Location(
+                region=region, country=country, division=div, location=loc
+            )
+            group.default_tree_location = tree_location
         builder = builder_factory(build_type, group, template_args, **context)
         builder.write_file(builds_file_fh)
 
         print("Wrote output files!")
         print(json.dumps(context))
+        session.rollback()  # Don't save any changes to the DB.
```

Comment:
why arn't we saving changes to the DB?

Reply:
This script is just meant to be a quick and dirty way to generate lots of different nextstrain build.yaml files without too much setup, and internally it can associate groups with different locations than they normally use -- we don't want to write those re-associations to the db when we're just trying to generate some test output.